custom_card.json

```diff
@@ -0,0 +1,10 @@
+{
+  "color": "red",
+  "actions": {
+    "skip": true,
+    "reverse": true,
+    "swap": true,
+    "change color": true
+  },
+  "draw penalty": 2
+}
```

Comment:
I think it would be very tedious to write each identical card in the JSON. What if there are multiple identical cards (ex I think in official UNO there are four +4 cards...), is there a way to encode that easily without duplicating too much in the JSON?

Reply:
I guess we can write the preset cards in the card file?